CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 # Changelog
 
+## 0.0.1-alpha.32
+
+Released on 2025-12-05.
+
+### LSP server
+
+- Add modules to auto-import completions ([#21799](https://github.com/astral-sh/ruff/pull/21799))
+- Always register rename provider if client doesn't support dynamic registration ([#21789](https://github.com/astral-sh/ruff/pull/21789))
+- Support re-exports in auto-import ([#21779](https://github.com/astral-sh/ruff/pull/21779))
+- Support renaming import aliases ([#21792](https://github.com/astral-sh/ruff/pull/21792))
+
+### Core type checking
+
+- Complete support for `ParamSpec` ([#21445](https://github.com/astral-sh/ruff/pull/21445))
```

Comment:
I would just say

```suggestion
- Support `ParamSpec` ([#21445](https://github.com/astral-sh/ruff/pull/21445))
```

since our support before now wasn't really there at all, but there are still some more advanced `ParamSpec` features such as `Concatenate` that we don't yet support